Interview/AutomationExercise.md

```diff
@@ -0,0 +1,70 @@
+<p align="center">
+<img src="../logo.png">
+</p>
+
+#Babylon 
+# iOS Automation engineer technical exercise
+
+At babylon, we firmly believe that **transparency** is a core value that should be present in everything we do, and hiring the best person should reflect that belief. That is why defined a clear and open technical exercise to be completed by candidates applying for an iOS automation engineer role. 
+
+## Our expectations
+
+Babylon aims to hire only the most qualified people. That doesn't mean that a candidate must have several academic degrees or years of experience. We value **potential** equally as much as **knowledge** and **experience**. Above all, we value one's ability to **think independently**, **solve problems** and to **engage with colleagues**.
+
+As such, these are the main areas the team will focus our assessment of candidates on:
+
+- Technical expertise
+ - Languages (Swift, Java) and platforms (iOS)
+ - Frameworks (UI testing with XCTest)
+ - Knowledge of software testing practises (Manual and Automated)
+ - Automation Architectural design patterns (POM, Cucumber)
+ - Knowledge of iOS APIs
+ - Knowledge of the development cycle
```

Comment:
Pls add more spaces to enable indentation.

Reply:
Odd locally the indentation was fine